src/planscape/plan/models.py

```diff
@@ -82,16 +79,39 @@ class Scenario(models.Model):
     # we want alpha users to not be signed in.
     owner = models.ForeignKey(
         User, on_delete=models.CASCADE, null=True)  # type: ignore
+    
+    plan = models.ForeignKey(Plan, on_delete=models.CASCADE, null=True)  # type: ignore
+
+    # The creation time of the project, automatically set when the project is created.
+    creation_time: models.DateTimeField = models.DateTimeField(
+        null=True, auto_now_add=True)
 
     project = models.ForeignKey(
```

Comment:
Again to double-check: does Project correspond to ForsysParams in the db graph?

Reply:
yes mostly, although I'm not sure if that's changing given our discussion today. should we chat about it?

Reply:
lol. May we chat about it after the product vision for configs and scenarios has settled more?

(though if there's a sense we can get consensus between us and can convince everyone else it's the right way forward, maybe we can chat about it first?)

In the meantime, I don't want to block interfacing work and will go with what you're implementing.